.azure-pipelines/resources/chaincode/commercialpaper/commercial-paper/paper.go

```diff
@@ -0,0 +1,138 @@
+// Copyright the Hyperledger Fabric contributors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package commercialpaper
+
+import (
+	"encoding/json"
+	"fmt"
+
+	ledgerapi "github.com/hyperledger/fabric-chaincode-integration/commercialpaper/ledger-api"
```

Comment:
why am I looking at ledger-api here?

Reply:
That is a package that exists in the commercial paper sample. I copied what exists in the fabric-samples commercial paper Go